bin/setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This looks much better now. I am kind of wondering whether we should include the handy comments we have in `.env.example` as they describe things too. 

But rather than duplicating them in the setup script, we should probably copy the example and populating the values accordingly...... thoughts?

Reply:
maybe it's overkill, but would like to hear what others think.

Reply:
pls check latest